docs/extensions/TextBoxRegex.md

```diff
@@ -11,30 +11,24 @@ The [TextBoxRegex Property](https://docs.microsoft.com/dotnet/api/microsoft.tool
 
 The developer adds a regular expression to validate the TextBox Text against the regular expression throw Regex property or from selecting ValidationType property on the TextBox.
 
-The validation has 2 modes (`ValidationMode`):
+The validation has 3 modes (`ValidationMode`):
 1) Normal (Default) : This type will set TextBox IsValid attached property to false or true whether the TextBox text is a valid or not against the Regex property.
-2) Dynamic : This type extends 1) Normal and if is the newest input of the Textbox  is invalid, the character which is invalied will be deleted.
-
-
-Main Attached Properties:
```

Comment:
Why was this deleted?

Reply:
Thought this was updated in the new version here? 
https://github.com/Microsoft/WindowsCommunityToolkit/blob/master/docs/extensions/TextBoxRegex.md#attached-properties
So I have deleted the old code.